### PR TITLE
Add a precaution note to the "from HAR file" bundle generation method

### PR DESCRIPTION
**Pull request by myrzakereyms**
As per issue #663 we're adding a small note to the bundle generation instructions from a HAR file.

### CODE REVIEW

File: go/bundle/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ These command-line flags are common to all the three options:
 
 #### From a HAR file
 
-One convenient way to generate HAR file is via Chrome Devtools. Navigate to "Network" panel, and right-click on any resource and select "Save as HAR with content".
+One convenient way to generate HAR file is via Chrome Devtools. Navigate to "Network" panel, and right-click on any resource and select "Save as HAR with content" (before saving, make sure that the "Disable cache" option is checked in the same "Network" panel).
```
Comment (irori) [inline go/bundle/README.md:43]: Checking "Disable cache" after loading resources does not have any effect to HAR generation. So the right procedure is:

1. Open DevTools, navigate to Network panel
2. Make sure "Disable cache" is checked
3. Reload the page
4. "Save as HAR with content"

File: go/bundle/README.md [hunk 2]
```diff
@@ -40,7 +40,12 @@ These command-line flags are common to all the three options:
 
 #### From a HAR file
 
-One convenient way to generate HAR file is via Chrome Devtools. Navigate to "Network" panel, and right-click on any resource and select "Save as HAR with content".
+One convenient way to generate HAR file is via Chrome Devtools:
+
+1. Open DevTools, navigate to Network panel.
+2. Make sure that "Disable cache" box is checked.
+3. Reload the page.
+4. Right click on any resource and select "Save as HAR with content".
```
Comment (irori) [inline go/bundle/README.md:48]: Add a blank line here

**Review by irori — APPROVED** [review]

**Comment by irori** [comment]
Thanks!

**Comment by hayatoito** [comment]
Just in curious, 

> irori marked as non substantive for IPR from ash-nazg.

Could you share how we can mark the PR as non substantive for IPR? I'd like to know how to do that.

**Comment by irori** [comment]
> Could you share how we can mark a PR as non substantive for IPR? I'd like to know how to do that.

You can do it from the Repository Manager page (e.g. https://labs.w3.org/repo-manager/pr/id/WICG/webpackage/665), accessible via the "Details" link of the `ipr` check.

**Comment by hayatoito** [comment]
Thanks! I didn't know that.

**Pull request merged**